siac/hostcmd.go

```diff
@@ -55,8 +56,7 @@ func hostconfigcmd(param, value string) {
 		// convert price to hastings/byte/block
 		p, ok := new(big.Rat).SetString(value)
 		if !ok {
-			fmt.Println("could not parse price")
-			return
+			dieNoError("Could not parse price")
```

Comment:
I don't see much value in prefacing errors with `Error:`. The usual Go style seems fine to me:
```go
$ siac host
Command failed: no response from daemon
```